ember-stargate/package.json

```diff
@@ -99,15 +105,15 @@
   "exports": {
     ".": "./dist/index.js",
     "./*": {
-      "types": "./dist/*.d.ts",
+      "types": "./declarations/*.d.js",
```

Comment:
I was wondering if the `*.d.js` is a typo.

In my addons, I wrote the following for `"."` and `"./*"`:

```json
"exports": {
  ".": {
    "types": "./declarations/index.d.ts",
    "default": "./dist/index.js"
  },
  "./*": {
    "types": "./declarations/*.d.ts",
    "default": "./dist/*.js"
  },
  "./addon-main.js": "./addon-main.cjs"
},
```